File: examples/node/automations/mailboxes/list.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

config({ path: '../../../.env.production' });

const reclient = new Retab();

const automations = await reclient.processors.automations.mailboxes.list({
  processor_id: "proc_o4dtLxizT0kDAjeKuyVLA",
});

console.log(JSON.stringify(automations, null, 2));

File: examples/python/automations/outlook/list.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automations = reclient.processors.automations.outlook.list(
    processor_id="proc_o4dtLxizT0kDAjeKuyVLA",
)

print(automations.model_dump_json(indent=2))
